contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/gstreamer/category.pxu

```diff
@@ -0,0 +1,3 @@
+unit: category
+id: gstreamer
```

Comment:
I'd prefer the category to be renamed as "video-codec"
Ref: https://en.wikipedia.org/wiki/Video_codec

Reply:
Update